api/views/yangSearch/yangSearch.py

```diff
@@ -50,38 +53,68 @@ class YangSearch(Blueprint):
 
 @bp.record
 def init_logger(state):
-    bp.logger = log.get_logger('yang-search', '{}/yang.log'.format(state.app.config.d_logs))
+    bp.logger = log.get_logger('yang-search', f'{state.app.config.d_logs}/yang.log')
 
 
 @bp.before_request
 def set_config():
-    global ac
-    ac = app.config
+    global app_config
+    app_config = app.config
 
 
-# ROUTE ENDPOINT DEFINITIONS
-
-
-# @bp.route('/grep_search', methods=['POST'])
+@bp.route('/grep_search', methods=['GET'])
+@cache.cached(query_string=True, timeout=GrepSearchCacheEnum.GREP_SEARCH_CACHE_TIMEOUT.value)
 def grep_search():
-    if not request.json:
-        abort(400, description='No input data')
-    body = request.json
-    organizations: list[str] = body.get('organizations', [])
-    search_string: str = body.get('search')
-    inverted_search: bool = body.get('inverted_search', False)
-    case_sensitive: bool = body.get('case_sensitive', False)
+    if not (query_params := request.args):
+        abort(400, description='No query parameters were provided')
+    search_string: str = query_params.get('search')
     if not search_string:
         abort(400, description='Search cannot be empty')
+    organizations: list[str] = query_params.getlist('organizations')
+    inverted_search: bool = query_params.get('inverted_search', type=lambda v: v.lower() == 'true')
```

Comment:
This could be a `None` which would make the type hint not true. I guess this probably won't cause errors in the current code since it's still falsy, but it would be better to provide `False` as the default.